src/status_im/ui/screens/wallet/db.cljs

```diff
@@ -23,7 +23,7 @@
 ;; TODO(oskarth): Spec for prices as as: {:from ETH, :to USD, :price 290.11, :last-day 304.17}
 
 (defn- empty-amount? [amount]
```

Comment:
shouldn't just `empty?` work fine for both nil and "'?

Reply:
Good point